fixed-requirements.txt

```diff
@@ -7,12 +7,12 @@ chardet<3.1.0
 cffi<1.15.0
 # NOTE: 2.0 version breaks pymongo work with hosts
 dnspython>=1.16.0,<2.0.0
-cryptography==39.0.1
+cryptography==41.0.4
```

Comment:
`cryptography` 3.4.7 -> 39.0.1, `pyOpenSSL` 21.0.0 -> 23.1.0 and `paramiko` 2.10.5 -> 2.11.0  were just updated here https://github.com/StackStorm/st2/pull/6055 last week which fixed high and medium issues, but left one of low severity (leaving as is).

We can't update to cryptography `41` as that'll break the py3.6 support which is required for 3.8.1 patch version to be released. The team is going to drop py3.6 in the next st2 `v3.9.0` afterwards.


Please remove those changes and let's see what's left in terms of failing 🔴  tests.

---

I expect to see some difficulty with `gitpython` which also should support py3.6 somehow.

Reply:
Ah yes - I forgot to check that, I only tested against `py3.8` - I've dropped the changes to `cryptography` (and `pyOpenSSL` as well) now - let's see how that goes